src/WorkspaceDependencyViewExtension/Properties/Resources.resx

```diff
@@ -359,13 +359,22 @@
         jDoyqebiAAAAAElFTkSuQmCC
 </value>
   </data>
+  <data name="PackageHeaderText" xml:space="preserve">
+    <value>Package</value>
+  </data>
   <data name="ProvideFeedbackButton" xml:space="preserve">
     <value>Provide Feedback</value>
   </data>
   <data name="ProvideFeedbackError" xml:space="preserve">
     <value>Could not redirect to the Dynamo forum page for feedback:</value>
   </data>
+  <data name="RefreshButtonTooltipText" xml:space="preserve">
+    <value>Click to refresh package dependencies.</value>
```

Comment:
what do you think of 
`Click to recalculate workspace dependencies` - I know today we only show package dependencies so not sure if we should make this more general for the future.

Also refresh to me - is a bit ambiguous between the view of the dependencies or the actual dependencies on disk or the download state etc. - what do you think?